src/razor/src/completion/razorCompletionItemProvider.ts

```diff
@@ -86,6 +87,11 @@ export class RazorCompletionItemProvider extends RazorLanguageFeatureBase implem
             // In the code behind it's represented as __o = DateTime.
             const completionCharacterOffset = projectedPosition.character - hostDocumentPosition.character;
             for (const completionItem of completionItems) {
+                // vscode.CompletionItemKind is off by one compared to the LSP CompletionItemKind.
```

Comment:
Wow that is mighty annoying 😆

Reply:
![KermitWorriedGIF](https://github.com/dotnet/vscode-csharp/assets/475144/1961cc1d-5b42-4d42-9c87-6a4b58f136e0)